scripts/pdg.sh

```diff
@@ -77,6 +78,32 @@ main() {
             --output "${pdg}" \
         > "${test_dir}/pdg.log"
     )
+    (
+        export RUST_BACKTRACE=full # print sources w/ color-eyre
+        export RUST_LOG=error
+        export PDG_FILE="${pdg}"
+        local deps_dir="${CARGO_TARGET_DIR:-target}/${profile_dir_name}/deps"
+        local libc_dep="$(ls -1 --sort=time -r ${deps_dir}/liblibc-*.rlib | head -n1)"
+        cargo run \
+            --bin c2rust-analyze \
+            "${profile_args[@]}" \
+            -- \
+            --crate-name "${crate_name}" \
+            --edition 2021 \
+            --crate-type bin \
+            --sysroot $(rustc --print sysroot) \
+            "-C" \
+            "opt-level=3" \
+            "-C" \
+            "embed-bitcode=no" \
+            "--cfg" \
+            "feature=\"c2rust-analysis-rt\"" \
```

Comment:
This is specific to `c2rust instrument`.  We shouldn't need it for `c2rust-analyze` since we're not linking to the runtime.

```suggestion
```